express/code/scripts/scripts.js

```diff
@@ -227,9 +227,45 @@ if (new URLSearchParams(window.location.search).get('lingo')) {
 document.body.dataset.device = navigator.userAgent.includes('Mobile') ? 'mobile' : 'desktop';
 preDecorateSections(document);
 // LCP image decoration
+const eagerLoad = (img) => {
+  img?.setAttribute('loading', 'eager');
+  img?.setAttribute('fetchpriority', 'high');
+};
+
 (function decorateLCPImage() {
-  const lcpImg = document.querySelector('img');
-  lcpImg?.removeAttribute('loading');
+  const firstSection = document.querySelector('body > main > div:nth-child(1)');
+  if (!firstSection) return;
+
+  // Get all images in the first section
+  const images = firstSection.querySelectorAll('img');
+  if (images.length > 0) {
+    images.forEach(eagerLoad);
+
+    // Preload the first image to force early loading
+    const firstImg = images[0];
```

Comment:
I wonder if these are needed? I would assume that browser would already have enough hints and we won't need to create preload link tag?